pkg/controllers/managementuser/helm/controller.go

```diff
@@ -396,14 +397,14 @@ func (l *Lifecycle) writeKubeConfig(obj *v3.App, kubePath string, remove bool) e
 
 func isSame(obj *v3.App, revision *v3.AppRevision) bool {
 	if obj.Spec.ExternalID != "" {
-		if revision.Status.ExternalID == obj.Spec.ExternalID && reflect.DeepEqual(revision.Status.Answers, obj.Spec.Answers) && reflect.DeepEqual(revision.Status.ValuesYaml, obj.Spec.ValuesYaml) {
+		if revision.Status.ExternalID == obj.Spec.ExternalID && reflect.DeepEqual(revision.Status.Answers, obj.Spec.Answers) && reflect.DeepEqual(revision.Status.AnswersForceString, obj.Spec.AnswersForceString) && reflect.DeepEqual(revision.Status.ValuesYaml, obj.Spec.ValuesYaml) {
```

Comment:
`isSame` should consider `AnswersForceString`, but should there be any normalization performed to ensure only upgrading when needed?

For example ...
```
answersForceString1 = map[string]bool{}
answersForceString2 = map[string]bool{"test-key":false}
```
Should the above be considered to be changed, or because the key `"test-key"` returns false for both cases, should this be a no-op?

Reply:
> Should multi-cluster apps be included as well? I have that work in-progress, as it makes sense but could be moved to a separate effort.
>
 I suppose multi-cluster apps could be included.

> Should the above be considered to be changed, or because the key "test-key" returns false for both cases, should this be a no-op?
>
If you are ultimately comparing the same value, either way would be fine with me. 

What are your thoughts @dramich?

Reply:
Just following up on this. Do you have any comments on the above @dramich? (I'm not too familiar with this code)